build_scripts/build_usd.py

```diff
@@ -2270,6 +2270,13 @@ def ForceBuildDependency(self, dep):
 
 verbosity = args.verbosity
 
+if (sys.version_info[0] == 3 and sys.version_info[1] > 6) or sys.version_info[0] > 3:
+    # fix for git-bash on Windows using code page 1252 as default instead
+    # of unicode - functions like e.g. PrintCommandOutput fail writing
+    # unicode characters for filenames containing them
+    # `reconfigure` only exists in versions greater than Python 3.6
```

Comment:
What role does "reconfigure" play? Is there a spec we can link? Thanks!

Reply:
It's because the default encoding of `TextIOWrapper` is locale-specific (which is the type of object `sys.stdin` and `sys.stdout` are):

https://docs.python.org/3/library/io.html?highlight=reconfigure#io-text-encoding